core/src/main/java/com/linecorp/armeria/server/AbstractServiceBindingBuilder.java

```diff
@@ -175,6 +175,11 @@ public AbstractServiceBindingBuilder setHeaders(
         return this;
     }
 
+    public AbstractServiceBindingBuilder serviceErrorHandler(ServiceErrorHandler serviceErrorHandler) {
```

Comment:
You could also add this method to `ServiceConfigSetters` and add `@Override` here.